exporters/fluentd/CMakeLists.txt

```diff
@@ -14,20 +14,28 @@
 
 cmake_minimum_required(VERSION 3.12)
 
-project(opentelemetry-fluentd)
-
+# MAIN_PROJECT CHECK
+## determine if fluentd exporter is built as a subproject (using add_subdirectory) or if it is the main project
+##
+set(MAIN_PROJECT OFF)
```

Comment:
I understand, with `add_subdirectory` added inside the otel-cpp cmake file, it would get build just like any other exporter inside otel-cpp repo. And we don't have that support in otel-cpp ?

Reply:
In fact, we don't. However, elsewhere when we build a consolidated exporter for [Linux mdsd](https://github.com/Azure/fluentd-plugin-mdsd) -- we end up modifying the main project `CMakeLists.txt` anyways. What I'd like to show is how to add a 1-liner to build the main project _WITH_ fluentd exporter, in essence.

There is no change to current build. You can still build it as a standalone project as before. The new option is additive, not altering existing build behavior.